### PR TITLE
Add announcement style

### PR DESCRIPTION
**Pull request by lunson**
Follow up on yesterday's OME meeting: proposing a style for announcements on the home page that would get people's attention.

Copy used is for placeholder only. This requires real copy/content, please :)

For your review, @joshmoore.

### CODE REVIEW

**Comment by joshmoore** [comment]
Thanks, @lunson. I was too slow in responding to your trello notification. Let's continue the conversation here.

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <!-- begin banner announcement -->
         <a href="#" target="_blank">
             <section class="announcement">
-                <span><i class="fa fa-bullhorn"></i> IMPORTANT: This is an important announcement, yo. <span class="styled-link">Here is the link providing more details that ya’ll should read</span></span>
+                <span><i class="fa fa-bullhorn"></i> IMPORTANT: End-of-life for all Python 2-based versions of OMERO <span class="styled-link"><a href="{{ site.baseurl }}/2019/11/13/python-3-rollout.html">Read the Announcement</a> </span></span>
```
Comment (joshmoore) [inline index.html]: @lunson: this is currently looking pale

Screen Shot 2019-11-13 at 21 59 07

Comment (lunson) [reply index.html]: Gah! Looked fine on my local, sorry about that. This should be fixed now - white for maximum contrast.

**Comment by joshmoore** [comment]
Thanks, @lunson !
Screen Shot 2019-11-14 at 08 01 30
```
V
V
V
```
Screen Shot 2019-11-14 at 08 01 36

**Comment by sbesson** [comment]
Now deployed live to https://www.openmicroscopy.org/

**Pull request merged**